src/client.ts

```diff
@@ -162,14 +162,14 @@ export class RSPClient {
      * @param id unique identifier for the newly created server
      * @param timeout timeout in milliseconds
      */
-    createServerSync(pathOrBean: string | Protocol.ServerBean, id?: string, timeout: number = 2000): Promise<Protocol.ServerHandle> {
+    createServerSync(pathOrBean: string | Protocol.ServerBean, id?: string, attributes?: any, timeout: number = 2000): Promise<Protocol.ServerHandle> {
```

Comment:
Please add a type definition for attributes like in the createServerFrom* methods.